dcos/resource.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
So this is the new registry package builder config? Is this really a template?

Reply:
we need all the dcos catalog files.

Reply:
Ok.